tasks/tasks.go

```diff
@@ -33,6 +33,24 @@ const (
 	NOTFOUND
 )
 
+var DefaultTaskExecuter = NewTaskExectuer()
+
+func (s Status) String() string {
+	var str string
+
+	switch s {
+	case INPROGRESS:
+		str = "The task is in progress."
+	case SUCCESS:
+		str = "The task was successfully completed."
+	case FAILURE:
+		str = "The task failed."
+	case NOTFOUND:
+		str = "Error: not found"
+	}
+	return str
+}
+
 // NewTaskExectuer returns a TaskExecutor ready to execute.
```

Comment:
thought spelling was fixed in `task-queue` pr, but turns out not. `Executor` not `Exectuer`